.github/workflows/torchserve-nightly-build.yml

```diff
@@ -30,7 +32,29 @@ jobs:
       - name: Java Debugging
         run: |
           which javac
-      - name: Push nightly
+      - name: Push torchserve nightly
         env:
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+        # skip pushing PyPI nightly binaries for macOS & windows
+        if: matrix.os == 'ubuntu-latest'
         run: ts_scripts/push_nightly.sh torchserve-nightly
```

Comment:
the `push_nightly.sh` script has a call to `setup.py` which is not necessary. In fact we can get rid of that whole script.

I believe the pypi binaries will be available after running `binaries/build.py` so that's what we should upload with `twine`

Reply:
Good point. I addressed these. Now the yml file looks much simpler